packages/paste-core/components/corner-ornament/src/index.tsx

```diff
@@ -0,0 +1,10 @@
+export { CornerOrnament } from "./CornerOrnament";
+export { CornerOrnamentContainer } from "./CornerOrnamentContainer";
+export { CornerOrnamentElement } from "./CornerOrnamentElement";
+export type {
+  CornerOrnamentElementProps,
+  CornerOrnamentContainerProps,
+  CornerOrnamentProps,
+  CornerOrnamentType,
+  CornerOrnamentPosition,
```

Comment:
question: why are you exporting these 2?

Reply:
todo: remove